source/io/JavaScriptInvoke.cpp

```diff
@@ -46,10 +46,19 @@ VIREO_EXPORT const char* JavaScriptInvoke_GetParameterType(StaticTypeAndData *pa
     if (returnBuffer) {
         SubString typeName = parameterType->Name();
         returnBuffer->Append(typeName.Length(), (Utf8Char*)typeName.Begin());
+
+        if (parameterType->IsArray() && !parameterType->IsString()) {
```

Comment:
I need to check that it is not a string explicitly here for arrays because string types in vireo are also array types, but I only want to append the subtype to javascript arrays and not javascript strings.

Reply:
That's a good idea 👍